.docker/entrypoint/lib/commander.rb

```diff
@@ -50,40 +47,50 @@ def log(severity, message)
       end
 
       def execute_args(args)
-        command = args['<command>']
+        command = find_command(args)
         verify = args['--verify']
         dry_run = args['--dry-run']
-        jekyll_config = @jekyll_config_provider.provide(command)
-        execute_command(jekyll_config, command, dry_run, verify)
-        verify(jekyll_config) if verify
+        ignore_urls = args['--ignore-url']
+        log_level = args['--log-level']
+        jekyll_config_provider = JekyllConfigProvider.new(@jekyll_env, log_level)
```

Comment:
Codacy found an issue: [Line is too long. [81/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=5786435)